localforage/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Thanks. I followed the README so it points to the upstream package with the bundled types. Is that OK or should I totally remove the folder altogether?